README.md

```diff
@@ -7,15 +7,15 @@ Run [Django](https://www.djangoproject.com) projects from source using [Gunicorn
 #### Step 0: Get your Django project in shape
 There are a few ways that your Django project needs to be set up in order to be compatible with this Docker image.
 
-**setup.py**  
```

Comment:
eh sorry! Is there special meaning to the trailing characters? It seems to be doing this automatically.